### ISSUE TITLE
Compile error when using latest AppCompat version

### ISSUE THREAD

**Issue by michael-rapp** [michael-rapp/AndroidPreferenceActivity#9]
When using the library together with the latest version of the AppCompat v7 support library (22.1.1) the following compile error occurs:

```
"R.styleable.Toolbar_theme" cannot be resolved or is not a field.
on de.mrapp.android.preference.activity.view.ToolbarLarge#obtainTheme()
```

**Comment by michael-rapp** [comment]
Fixed in upcoming version 2.1.0.

**Comment by michael-rapp** [comment]
The upcoming release 2.1.0 will rely on AppCompat v7 revision 22. Because the library will then use the support library's new class `AppCompatActivity`, it will not be compatible with revision 11 anymore.
